src/main/java/org/bitsofinfo/hazelcast/discovery/docker/swarm/SwarmDiscoveryUtil.java

```diff
@@ -33,6 +33,9 @@
 import com.spotify.docker.client.messages.swarm.Task;
```

Comment:
please don't couple large formatting changes with tangible logic changes, makes it difficult to figure out whats actually changed.

Reply:
Understood.